src/Cradle.hs

```diff
@@ -16,8 +16,10 @@ module Cradle
     -- * Possible Inputs
     Input,
     StdinHandle (..),
+    NoStdin (..),
     StdoutHandle (..),
     StderrHandle (..),
+    DelegateCtrlC (..),
```

Comment:
There's no tests for `DelegateCtrlC` since I think it'd be a lot of work to test and we just hand it through as a boolean to `createProcess_`.